R/FilterStateLogical.R

```diff
@@ -213,10 +220,10 @@ LogicalFilterState <- R6::R6Class( # nolint
       invisible(NULL)
     },
     validate_selection = function(value) {
-      if (!(checkmate::test_logical(value, max.len = 1, any.missing = FALSE))) {
+      if (!(checkmate::test_logical(value, max.len = 2, any.missing = FALSE))) {
```

Comment:
because
```r
checkmate::test_logical(c(TRUE, TRUE), max.len = 2, any.missing = FALSE)
#> TRUE
```


```suggestion
      if (!(checkmate::test_logical(value, any.missing = FALSE, unique = TRUE))) {
```